src/libAtomVM/opcodesswitch.h

```diff
@@ -2960,6 +2960,11 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
 
                 TRACE("try_case/1, reg=%c%i\n", T_DEST_REG(dreg_type, dreg));
 
+                #ifdef IMPL_EXECUTE_LOOP
+                    //TODO: here just write to y registers is enough
+                    WRITE_REGISTER(dreg_type, dreg, term_nil());
```

Comment:
term_nil()? what the destination register should contain?

Reply:
I believe the purpose of `try_case` is to clear the exception on the stack pointed to by the destination register.  I admit this is a copy from OP_TRY_END